Exception/InvalidStateException.php

```diff
@@ -1,11 +0,0 @@
-<?php
```

Comment:
Could we keep everything at the root - so not in `src/`? That would keep it consistent, at least, with other bundles.